WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -701,8 +701,18 @@ private extension DashboardViewController {
 
     func onPullToRefresh() async {
         ServiceLocator.analytics.track(.dashboardPulledToRefresh)
-        await viewModel.syncAnnouncements(for: siteID)
-        await reloadDashboardUIStatsVersion(forced: true)
+        await withTaskGroup(of: Void.self) { group in
+            group.addTask { [weak self] in
+                guard let self = self else { return }
```

Comment:
nit:
```suggestion
                guard let self else { return }
```

Reply:
Done in c1a89434dc696502edf1c786e67653d5694fbea3